Update README.md (#2730)
treeverse/lakeFS: deployments/compose/README.md

# lakeFS `docker-compose`

> Compose file for lakeFS using variety of services lakeFS can work with. 

Reviewer: ```suggestion
Compose file for lakeFS using variety of services lakeFS can work with.
```

> Compose file for lakeFS using variety of services lakeFS can work with. 

Reviewer 2: can remove the extra space

Author: done

docker compose ps
```

> To access the lakeFS UI, go to http://localhost:8000. The setup stage is done for you, and your user name and passwords are AKIAIOSFODNN7EXAMPLE and wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY compatibly. 

Reviewer: ```suggestion
To access the lakeFS UI, go to http://localhost:8000. 
Your credentials are (The setup stage was done for you):
user:AKIAIOSFODNN7EXAMPLE
pass: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY 
```

Reviewer 2: chips

> To access the lakeFS UI, go to http://localhost:8000. The setup stage is done for you, and your user name and passwords are AKIAIOSFODNN7EXAMPLE and wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY compatibly. 

Reviewer 2: ```suggestion
To access the lakeFS UI, go to http://localhost:8000. The docker-compose also runs the setup step and configure admin user with the following credentials:

Access key ID: `AKIAIOSFODNN7EXAMPLE`
Secret access key: `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY` 
```

Author: done

Reviewer 2: UI uses the terms 'access key id' and 'secret access key' like S3.

Author: thanks, fixed.
